packages/runtime/package.json

```diff
@@ -37,7 +37,6 @@
   },
   "dependencies": {
     "@apidevtools/json-schema-ref-parser": "^9.0.6",
-    "@botpress/messaging-client": "1.1.5",
```

Comment:
Since you are using this library in the runtime package, I would keep it in its package.json